app/src/main/java/doit/study/droid/quiz/QuizPageViewModel.kt

```diff
@@ -0,0 +1,122 @@
+package doit.study.droid.quiz
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.google.android.gms.analytics.HitBuilders
+import com.google.android.gms.analytics.Tracker
+import doit.study.droid.BuildConfig
+import doit.study.droid.R
+import doit.study.droid.data.local.QuizDatabase
+import doit.study.droid.utils.AnalyticsData
+import doit.study.droid.utils.Event
+import timber.log.Timber
+import javax.inject.Inject
+
+
+class QuizPageViewModel @Inject constructor(
+private val quizDatabase: QuizDatabase,
+private val analyticsTracker: Tracker
+) : ViewModel() {
+    private val _item = MutableLiveData<QuizView>()
+    val item: LiveData<QuizView> = _item
+
+    private val _showToastForAnswer = MutableLiveData<Event<Int>>()
+    val showToastForAnswer: LiveData<Event<Int>> = _showToastForAnswer
+
+    private val _playSound = MutableLiveData<Event<Boolean>>()
+    val playSound: LiveData<Event<Boolean>> = _playSound
+
+    private val _commitButtonState = MutableLiveData<Int>()
+    val commitButtonState: LiveData<Int> = _commitButtonState
+
+    private val _lockInteraction = MutableLiveData<Unit>()
+    val lockInteraction: LiveData<Unit> = _lockInteraction
+
+    private val _showToastForEvaluation = MutableLiveData<Event<Int>>()
+    val showToastForEvaluation: LiveData<Event<Int>> = _showToastForEvaluation
+
+    private val _swipeToTheNextPage = MutableLiveData<Event<Unit>>()
```

Comment:
All of these doesn't sound like a data, but like method names. It would be more clear to name it respectively: `toastEvent`, `soundEvent`, etc

Reply:
Will be fixed in https://github.com/mgolokhov/dodroid/issues/139